src/rest/APIRequest.js

```diff
@@ -18,8 +18,9 @@ class APIRequest {
 
     let queryString = '';
     if (options.query) {
-      // Filter out undefined query options
-      const query = Object.entries(options.query).filter(([, value]) => value !== null && typeof value !== 'undefined');
+      let query = (options.query instanceof URLSearchParams ? [...options.query] : Object.entries(options.query))
```

Comment:
```suggestion
      const query = (options.query instanceof URLSearchParams ? [...options.query] : Object.entries(options.query))
```
i don't see this variable being re-assigned anywhere so it can be a const